### DISCUSSION TITLE
Gear door bug

### DISCUSSION THREAD

**Discussion by PlaneHaven** [magknight/787aviatorsTracker#841] [Completed things]
Fix landing gear doors opening while rotating, it opens randomly during a takeoff run without actually putting the gear up

**Comment by aeroniemi** [comment]
does it happen any other time than during liftoff?

**Reply by PlaneHaven** [reply]
Only lift off

**Reply by aeroniemi** [reply]
that's realistic - watch a video of a dash9 taking off and you'll see that they open about 1 second after liftoff, way before the gear handle is moved

**Reply by PlaneHaven** [reply]
Ok well thats a dash 9 not a 787 I do not see any 787 do that 1 second after lift off

**Reply by aeroniemi** [reply]
the dash9 is a 787-9...

**Reply by PlaneHaven** [reply]
oh wait lol my bad i thought it was de haviland dash 9 LOL

**Reply by PlaneHaven** [reply]
But this one is way too early

**Reply by aeroniemi** [reply]
is it happening about 1 second after liftoff?

**Reply by PlaneHaven** [reply]
wait nvm its the perfect timing as the real life one
